src/pseudo_intrinsics.h

```diff
@@ -490,35 +555,14 @@ typedef __m64i vtype;
 #endif /* __SSE3__ || __MIC__ */
```

Comment:
You mean the one "for really old CPUs"? I'm not sure of its origin but I guess it makes sense. Anything that can save a single CPU cycle is...